cmd/vic-machine/create/create.go

```diff
@@ -379,8 +379,37 @@ func (c *Create) processContainerNetworks() error {
 		c.MappedNetworksIPRanges[vicnet] = pools[vnet]
 		c.MappedNetworksDNS[vicnet] = dns[vnet]
 
+		delete(gws, vnet)
+		delete(pools, vnet)
+		delete(dns, vnet)
 	}
 
+	var hasError bool
+	if len(gws) > 0 {
+		log.Errorf("Following container network gateway is set, but CONTAINER-NETWORK not found. Please check your --container-network and --container-network-gateway")
```

Comment:
Should be: "The following container network gateway is set, but CONTAINER-NETWORK cannot be found. Please check the --container-network and --container-network-gateway settings"